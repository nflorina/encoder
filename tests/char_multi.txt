aaaccce
b
1
END
