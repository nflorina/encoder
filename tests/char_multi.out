1 1 0
aaccceaabe
aaccceaabe
